### PR TITLE
Add ValleyPvP Network

### PR DESCRIPTION
**Pull request by ByJoako**
### General:

* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:

* [x] My formatting matches the pre-existing formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping with my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:

* [x] My image is a `.png` file.
* [x] I uploaded the image in the `logos` directory.
* [x] My image is transparent & square (1:1 aspect ratio).
* [x] My image is a minimum of `512` pixels in width and height.
* [x] My images image name matches its mapping id.

### CODE REVIEW

**Review by GrowlyX — DISMISSED** [review]

**Comment by GrowlyX** [comment]
@ByJoako, please resolve these conflicts:
image

**Comment by ByJoako** [comment]
I think it was solved

**Review by GrowlyX — APPROVED** [review]

File: servers.json [hunk 1]
```diff
@@ -1662,5 +1662,14 @@
 	"addresses": [
 	    "megaplanet.net"
 	]
+    },
+  {
+
```
Comment (GrowlyX) [inline servers.json:1667]: Could you remove this indent for me?

File: servers.json [hunk 2]
```diff
@@ -1662,5 +1662,14 @@
...
+  {
+    
+	"id": "valleypvp",
+	"name": "ValleyPvP Network",
+	"addresses": [
+	    "valleypvp.us"
+  ]
     }
 ]
+
```
Comment (GrowlyX) [inline servers.json:1675]: This one as well

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Review by GrowlyX — APPROVED** [review]

**Comment by GrowlyX** [comment]
Thanks!

**Pull request merged**